.github/workflows/docker_development_image.yml

```
@@ -0,0 +1,37 @@
+name: One App Development Image Build and Push
+on:
+  repository_dispatch:
+    types: [one-app-docker-dev-build]
+jobs:
+  one-app-docker-dev-build:
+    name: One App Development Image Build and Push
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Docker login
+        run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login --username ${{ secrets.DOCKER_USER }} --password-stdin
+      - name: Build development docker image
+        run: docker build -t dev . --target=development
+      - name: Tag Docker Images
+        run: |
+          docker tag dev ${{ secrets.DOCKER_USER }}/one-app-dev:${{ github.event.client_payload.major_version }}
+          docker tag dev ${{ secrets.DOCKER_USER }}/one-app-dev:${{ github.event.client_payload.major_version}}.${{ github.event.client_payload.minor_version }}
+          docker tag dev ${{ secrets.DOCKER_USER }}/one-app-dev:${{ github.event.client_payload.docker_tag_version }}
+          docker tag dev ${{ secrets.DOCKER_USER }}/one-app-dev:latest
+      - name: Push Docker Images
+        run: |
+          docker push ${{ secrets.DOCKER_USER }}/one-app-dev:${{ github.event.client_payload.major_version}}
+          docker push ${{ secrets.DOCKER_USER }}/one-app-dev:${{ github.event.client_payload.major_version}}.${{ github.event.client_payload.minor_version }}
+          docker push ${{ secrets.DOCKER_USER }}/one-app-dev:${{ github.event.client_payload.docker_tag_version}}
+          docker push ${{ secrets.DOCKER_USER }}/one-app-dev:latest
+      - name: Trigger Image Publishing
```

```suggestion
      - name: Publish One App Statics
```
Does not appear to be publishing an image